### PR TITLE
[MNT-22325] Copy Keystore to installation info for 7.x

### PR DESCRIPTION
**Pull request by gravitonian**

### CODE REVIEW

File: content-services/7.0/install/zip/tomcat.md [hunk 1]
```diff
@@ -217,6 +217,33 @@ After you've extracted the Content Services distribution zip, several directorie
...
+        * `<TOMCAT_HOME>/alf_data/keystore/metadata-keystore/keystore`
+        * `<TOMCAT_HOME>/alf_data/keystore/metadata-keystore/keystore-passwords.properties`
+
+2. Configure Tomcat 9 to use the default keystore files
+
+    1. Open `<TOMCAT_HOME>/bin/catalina.bat` in a text editor.
+
+    2. Add the following line to `catalina.bat`:
+
+       `set “JAVA_TOOL_OPTIONS=-Dencryption.keystore.type=JCEKS -Dencryption.cipherAlgorithm=DESede/CBC/PKCS5Padding -Dencryption.keyAlgorithm=DESede -Dencryption.keystore.location=<TOMCAT_HOME>/alf_data/keystore/metadata-keystore/keystore -Dmetadata-keystore.password=mp6yc0UD9e -Dmetadata-keystore.aliases=metadata -Dmetadata-keystore.metadata.password=oKIWzVdEdA -Dmetadata-keystore.metadata.algorithm=DESede”`
```
Comment (anxumalo) [inline content-services/7.0/install/zip/tomcat.md:243]: Can we use a codeblock here so it's easier for customers to copy the setting? (also applies to previous version)

**Review by anxumalo — COMMENTED** [review]
Builds OK 👍🏼. Looks good @gravitonian - one minor suggestion.

File: content-services/7.0/install/zip/tomcat.md (hunk 1 above)
Comment (gravitonian) [reply content-services/7.0/install/zip/tomcat.md]: Done

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**